.github/workflows/build_pytorch_yolo.yml

```diff
@@ -0,0 +1,69 @@
+name: Pytorch Yolo Detection
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  push:
+    branches: [master]
+    paths:
+      - ".github/workflows/build_pytorch_yolo.yml"
+      - "pytorch-yolo-image/**"
+      - "scripts/install_libtorch.sh"
+  pull_request:
+    branches: [master]
+    paths:
+      - ".github/workflows/build_pytorch_yolo.yml"
+      - "pytorch-yolo-image/**"
+      - "scripts/install_libtorch.sh"
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build_openvino_examples:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        rust: [1.71]
+    container:
+      image: wasmedge/wasmedge:ubuntu-build-clang
+    steps:
+      - name: Checkout Wasi-NN examples
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Install Rust-stable
+        uses: dtolnay/rust-toolchain@stable
+        with:
+          toolchain: ${{ matrix.rust }}
+          target: wasm32-wasi
+
+      - name: Install dependencies
+        run: |
+          apt update
+          apt install -y libtbb2
+
+      - name: Install LibTorch
+        working-directory: scripts
+        run: |
+          bash install_libtorch.sh
+
+      - name: Install WasmEdge
+        env:
+          CMAKE_BUILD_TYPE: "Release"
+          VERSION: "0.13.4"
+        run: |
+          curl -sSf https://raw.githubusercontent.com/WasmEdge/WasmEdge/master/utils/install.sh | bash -s -- -v $VERSION -p /usr/local
```

Comment:
@Charles-Schleich Can you trigger the CI action on this PR branch so that we can make it passes all CI before merging? Thanks.

Reply:
@juntao Sure thing, 
The CI was failing, i've pushed a fix as of 2 minutes ago.

I'm debating about committing the nano YOLO model  to the repository for testing. 
Pros: we dont need the CI step of having to download the model.
Cons: it bloats the repo when cloning by ~13MB. 

I don't expect the model to change and i'm not testing any sort of output from the example, 
purely that it can build, and run inference without failing / crashing.

Reply:
Hi @Charles-Schleich i think you need to probably also include the Pytorch plugin installation in the workflow when installing WASMEDGE